extra_data/file_access.py

```diff
@@ -320,14 +324,19 @@ def _read_data_sources(self):
                 # TODO: Do something with groups?
             elif category == 'CONTROL':
                 control_sources.add(h5_source)
+            elif category == 'REDUCTION':
+                reduction_info.add(h5_source)
+            elif category == 'ERRATA':
+                errata.add(h5_source)
             elif category == 'Karabo_TimerServer':
                 # Ignore virtual data source used only in file format
                 # version 1.1 / pclayer-1.10.3-2.10.5.
                 pass
             else:
                 raise ValueError("Unknown data category %r" % category)
```

Comment:
So this bit is probably the big sore point for this proposal - pretty much *all* prior versions of EXtra-data (or even `karabo_data` for that matter) will break with these new files.

Since this only affects the `METADATA/dataSources` entry but not the actual existance of the root groups, one way around this would be not list these sources here and either:
* Not list them at all and instead discover them by iterating over the root group
* List them in a distinct dataset only for these virtual metadata sources
* List both the *real* and *virtual* sources in a new dataset while preserving the old one for compatibility

Reply:
Yes, I don't see any way around it though, except if we add below METADATA after all. Generally one should possibly make this less restrictive in my view, as it could hit us again in the future.